starboard/BUILD.gn

```diff
@@ -296,5 +292,24 @@ if (current_toolchain == starboard_toolchain) {
     }
   }
 }
-# TODO: Create a starboard_unittest target here and fold any gtest_target_type
-# in it, e.g. common:common_test.
+
+test("starboard_unittests") {
```

Comment:
Did you try running this? I can't remember offhand if the test template includes a main entry point or if you still need to include a source/dep that defines it

Reply:
Yes, I did on Linux and as the description says it run 105 tests of which one stubbornly failed.